pos_payment_change/wizards/view_pos_payment_change_wizard.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+Copyright (C) 2015-Today GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <record id="view_pos_payment_change_wizard_form" model="ir.ui.view">
+        <field name="model">pos.payment.change.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <group col="4">
+                    <field name="order_id" />
+                    <field name="amount_total" invisible="1" />
+                    <field name="old_line_ids" colspan="4">
+                        <tree>
+                            <field name="old_payment_method_id" />
```

Comment:
```suggestion
                            <field name="old_payment_method_id" widget="selection"/>
```

that should do the job, regarding @victor-champonnois remarks.

Reply:
I have added the changes that you have requested in the PR, thanks